src/Utils.ts

```diff
@@ -136,4 +137,48 @@ export module Utils {
             return vscode.workspace.getConfiguration("spring.initializr").get<string>("serviceUrl");
         }
     }
+
+    export async function getTargetPomXml(): Promise<vscode.Uri> {
+        if (vscode.window.activeTextEditor) {
+            const fsPath: string = vscode.window.activeTextEditor.document.fileName;
+            if ("pom.xml" === path.basename(fsPath).toLowerCase()) {
+                return vscode.Uri.file(fsPath);
+            }
+        }
+
+        const candidates: vscode.Uri[] = await vscode.workspace.findFiles('**/pom.xml');
+        if (candidates && candidates.length > 0) {
```

Comment:
Since we have lodash in dependency, simply leveraging it to check the array is empty or not